.github/workflows/link-checker.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
       - name: Link Checker
         id: lychee
-        uses: lycheeverse/lychee-action@ec3ed119d4f44ad2673a7232460dc7dff59d2421 # ratchet:lycheeverse/lychee-action@v1.8.0
+        uses: lycheeverse/lychee-action@22134d37a1fff6c2974df9c92a7c7e1e86a08f9c # ratchet:lycheeverse/lychee-action@v1.9.0
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
```suggestion
```

Reply:
New version includes the github token automatically; cf. https://github.com/lycheeverse/lychee-action/commit/7cdbd6424d9567a3b47f5928d476b0ce10cb9da9